advocacy_docs/edb-postgres-ai/cloud-service/references/distributed_high_availability/pgd_cli_ba.mdx

```diff
@@ -48,70 +49,93 @@ Without a properly configured `.pgpass`, you receive a database connection error
 To use the PGD CLI with your database connection string, use the `--dsn` flag with your PGD CLI command:
 
 ```bash
-pgd show-nodes --dsn "<your_connection_string>"
+pgd nodes list --dsn "<your_connection_string>"
 ```
 
 ## PGD commands in Cloud Service
 
-!!!Note
-
-Three EDB Postgres Distributed CLI commands don't work with distributed high-availability Cloud Service clusters: `create-proxy`, `delete-proxy`, and `alter-proxy-option`. These commands are managed by Cloud Service, as Cloud Service runs on Kubernetes. It's a technical best practice to have the Kubernetes operator handle these functions.
-!!!
-
 The examples that follow show the most common PGD CLI commands with a Cloud Service cluster.
 
-### `pgd check-health`
+### `pgd cluster show --health`
 
-`pgd check-health` provides statuses with relevant messaging regarding the clock skew of node pairs, node accessibility, the current raft leader, replication slot health, and versioning consistency:
+`pgd cluster show --health` provides statuses with relevant messaging regarding the clock skew of node pairs, node accessibility, the current raft leader, replication slot health, and versioning consistency:
 
 ```
-$ pgd check-health --dsn "postgres://edb_admin@p-mbx2p83u9n-a.pg.biganimal.io:5432/bdrdb?sslmode=require"
+pgd cluster show --health --dsn "postgres://edb_admin@p-mbx2p83u9n-a.pg.biganimal.io:5432/bdrdb?sslmode=require"
 __OUTPUT__
```

Comment:
This command is now a tabular by default. Please update the output or add the `-o markdown` to the command if the report format output is expected.